game/templates/game/rules.html

```
@@ -655,7 +666,7 @@ <h1 class="header-subtitle">Interactive Chess Rulebook</h1>
         <div class="rule-nav-item" onclick="showRule('pieces')" id="nav-pieces">
             <span class="icon">♟</span> Piece Moves
         </div>
-        <button class="rule-nav-item" type="button" onclick="showRule('values')" id="nav-values">
+        <button class="rule-nav-item" onclick="showRule('values')" id="nav-values">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)